examples/external-data/src/mock-customer-service/service.ts

```diff
@@ -263,10 +310,13 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 			});
 		}
 
+		const containerUrl = { TenantId: tenantId, DocumentId: documentId };
```

Comment:
`containerUrl -> containerSessionInfo`? Also in the logged message below, and in line 233 where we loop over them.